pulumi/infra/network.py

```diff
@@ -49,7 +49,7 @@ def __init__(
             cidr_block=str(cidr_block),
             enable_dns_hostnames=True,
             enable_dns_support=True,
-            tags={"Name": f"{name}-{DEPLOYMENT_NAME}"},
+            tags={"Name": f"{name}"},
```

Comment:
This was done so that it was easier to do vpc peering

Reply:
How so?